api/helpers/uploadedFileErrorChecks.js

```diff
@@ -37,48 +44,97 @@ function initialiseClamScan(req) {
     } = req._sails.config.upload;
     const clamAvOptions = {
         debugMode: JSON.parse(clamavDebugEnabled) || false,
-        clamscan: {active: false},
+        clamscan: { active: false },
         clamdscan: {
             active: false,
             host: clamavHost,
             port: clamavPort,
         },
     };
-
-    return new NodeClam().init(clamAvOptions);
+    try {
+        return new NodeClam().init(clamAvOptions);
+    } catch (err) {
+        throw new Error(err);
+    }
```

Comment:
also check slack for more detail on this. the error will just throw from `NodeClam().init(clamAvOptions);` - so unless you are doing something with the error (like changing the message) there's not too much reason to rethrow it manually. Are you just trying to make it clear that it throws?

Reply:
Hmmm, my assumption was that if you use async/await you need to put the await function in a try catch or you'll get an unhandled promise error if there's an error 🤔 

Actually you're right, the parent function is taking care of handling the errors 🤦 
I'll get that fixed.